### ISSUE TITLE
💡 [REQUEST] - <title>学习率不改变，有人知道吗？

### ISSUE THREAD

**Issue by JiangnanRomeo** [QwenLM/Qwen-VL#397]
### 摘要 | Summary

1

### 基本示例 | Basic Example

{'loss': 1.1777, 'learning_rate': 1e-05, 'epoch': 0.79}                                                                                                                                                                                             
{'loss': 1.0297, 'learning_rate': 1e-05, 'epoch': 0.8}                                                                                                                                                                                              
{'loss': 1.0975, 'learning_rate': 1e-05, 'epoch': 0.81}                                                                                                                                                                                             
{'loss': 1.0213, 'learning_rate': 1e-05, 'epoch': 0.82}                                                                                                                                                                                             
{'loss': 0.9937, 'learning_rate': 1e-05, 'epoch': 0.83}                                                                                                                                                                                             
{'loss': 0.979, 'learning_rate': 1e-05, 'epoch': 0.84}                                                                                                                                                                                              
{'loss': 0.9382, 'learning_rate': 1e-05, 'epoch': 0.85}                                                                                                                                                                                             
{'loss': 0.9343, 'learning_rate': 1e-05, 'epoch': 0.86}                                                                                                                                                                                             
{'loss': 0.8786, 'learning_rate': 1e-05, 'epoch': 0.87}                                                                                                                                                                                             
{'loss': 0.9547, 'learning_rate': 1e-05, 'epoch': 0.88}                                                                                                                                                                                             
{'loss': 0.9132, 'learning_rate': 1e-05, 'epoch': 0.89}                                                                                                                                                                                             
{'loss': 0.8164, 'learning_rate': 1e-05, 'epoch': 0.9}                                                                                                                                                                                              
{'loss': 0.8481, 'learning_rate': 1e-05, 'epoch': 0.91}                                                                                                                                                                                             
{'loss': 0.7252, 'learning_rate': 1e-05, 'epoch': 0.92}                                                                                                                                                                                             
{'loss': 0.7812, 'learning_rate': 1e-05, 'epoch': 0.93}                                                                                                                                                                                             
{'loss': 0.7753, 'learning_rate': 1e-05, 'epoch': 0.94}                                                                                                                                                                                             
{'loss': 0.7889, 'learning_rate': 1e-05, 'epoch': 0.95}                                                                                                                                                                                             
{'loss': 0.734, 'learning_rate': 1e-05, 'epoch': 0.96}                                                                                                                                                                                              
{'loss': 0.7014, 'learning_rate': 1e-05, 'epoch': 0.97}                                                                                                                                                                                             
{'loss': 0.6886, 'learning_rate': 1e-05, 'epoch': 0.98}                                                                                                                                                                                             
{'loss': 0.6154, 'learning_rate': 1e-05, 'epoch': 0.99}                                                                                                                                                                                             
{'loss': 0.677, 'learning_rate': 1e-05, 'epoch': 1.0}

### 缺陷 | Drawbacks

1

### 未解决问题 | Unresolved questions

1
